buildtest/buildsystem/base.py

```diff
@@ -787,7 +787,7 @@ def simple_run(self, workdir=None):
         """
 
         if workdir:
-            return [f"{workdir}/{self.executable}"]
+            return [os.path.join(workdir / self.executable)]
```

Comment:
Why not `os.path.join(workdir, self.executable)` ?

Reply:
the reason why i put as a list is in future there could be different types of run cmd right now its a single line, as a standalone executable or running executable with arguments. Running test via mpi, or debugging and profiling code could be multiline commands that  are post build. It would be easier to add items to list, and the ``build_run_cmd`` could be expanded to encapsulate different run scenarios. There could be a combination of YAML and similar to ``run`` key from ``script`` to allow for more customized tests.

Reply:
Sorry I wasn’t asking about the list - I’m asking about the /

Reply:
My thought was to allow flexibility in future if user want's to run a test not relative to their working directory, they can provide a relative directory. For example, building a binary with different flags  needs to have a unique executable name, user could have way to organize where executable is generated and other artifacts as a result of execution of test.

Right now it's possible two sections in Buildspec that reference same sourcefile generate the same executable. This is because the default name of executable is related to source file. One possibility is to have a key for user to specify name of executable, but that is unrelated to your comment.

Reply:
I’ll try again - why did you use a slash (/) instead of an expected comma for two args in is.path.join?